src/pages/backCompat/signIn.vue

```diff
@@ -52,7 +52,9 @@ const penpalMethods = {
   isLoginAvailable: (kind: SocialLoginType) =>
     availableLogins.value.includes(kind),
   getPublicKey: handleGetPublicKey,
-  getAvailableLogins: () => [...availableLogins.value],
+  getAvailableLogins: () => [
+    ...availableLogins.value.filter((item) => item !== 'firebase'),
```

Comment:
Why don't you directly return the filtered array?

Reply:
sure, will do that